rsts/concepts/execution_timeline.rst

```diff
@@ -19,34 +21,38 @@ The illustration above refers to a simple workflow, with 2 nodes N1 & N2. This c
-the latency involves, time it takes to,
-#. Gather outputs for a node after the node completes
-#. Send an observation event to FlyteAdmin. Failing to do so is regarded as an error and will be tried until successful or we exhaust system max retries. (as a default the the max system retries is configured to be 30 and can be altered per deployment)
-#. Persist data to Kubernetes
-#. Receive the persisted object back from Kubernetes (as this process is eventually consistent using informer caches)
-#. Gather inputs for a node before the node starts
-#. Send a queued event for the next node to FlyteAdmin (this is what is persisted and drives the UI/CLI and historical information)
+Transition latency refers to the time between successive node executions, i.e between N1 & N2. For the first node (N1) this latency also encapsulates executing the ``Start node``. 
 
-**This is one of the latencies that the core team is working on optimizing**
+Similarly, the last node also encapsulates executing ``End node`` as well. ``Start Node`` and ``End Node`` are capstones inserted to mark the beginning and end of the DAG.
+
+The Latency involves time consumed to:
+
+#. Gather outputs for a node after the node completes.
+#. Send an observation event to FlyteAdmin. Failing to do so will be regarded as an error and will be tried until it is a success or we exhaust system max retries. (as a default the the max system retries is configured to be 30 and can be altered per deployment).
+#. Persist data to Kubernetes.
+#. Receive the persisted object back from Kubernetes (as this process is eventually consistent using informer caches).
+#. Gather inputs for a node before the node starts.
+#. Send a queued event for the next node to FlyteAdmin (this is what is persisted and drives the UI/CLI and historical information).
+
+**This is one of the latencies that the core team is working on optimizing!**
 
 Queuing Latency
 ================
-This is the time it takes for Kubernetes to start the pod, or the other service to start the job or an HTTP throttle to be met or any rate-limiting to be overcome. This
-is usually tied to available resources and quota and is out of control for Flyte
+This is the time taken by Kubernetes to start the pod, or other services to start the job or an HTTP throttle to be met or any rate-limiting that needs to be overcome. This
+is usually tied to the available resources and quota and is out of control for Flyte.
 
 Completion Latency
 ===================
-Time it takes to mark the workflow as complete and accumulate outputs of a workflow after the last node has completed.
+The time taken to mark the workflow as complete and accumulate outputs of a workflow after the last node completes its execution.
 
-**This is one of the latencies that the core team is working on optimizing**
+**This is one of the latencies that the core team is working on optimizing!**
```

Comment:
Can we just have a note to say what all latencies we are working on optimizing rather than writing this line again and again?